Source/MethodCall.cs

```diff
@@ -416,51 +385,67 @@ public IVerifies AtMost(int callCount)
 		protected IVerifies RaisesImpl<TMock>(Action<TMock> eventExpression, Delegate func)
 			where TMock : class
 		{
-			var ev = eventExpression.GetEvent((TMock)Mock.Object);
-			this.mockEvent = ev.MemberInfo;
-			this.mockEventArgsFunc = func;
+			var ev = eventExpression.GetEvent((TMock)mock.Object).MemberInfo;
+			if (ev != null)
+			{
+				this.raiseEventResponse = new RaiseEventResponse(this.mock, ev, func, null);
+			}
+			else
+			{
+				this.raiseEventResponse = null;
+			}
+
 			return this;
 		}
 
 		protected IVerifies RaisesImpl<TMock>(Action<TMock> eventExpression, params object[] args)
 			where TMock : class
 		{
-			var ev = eventExpression.GetEvent((TMock)Mock.Object);
-			this.mockEvent = ev.MemberInfo;
-			this.mockEventArgsParams = args;
+			var ev = eventExpression.GetEvent((TMock)mock.Object).MemberInfo;
+			if (ev != null)
+			{
+				this.raiseEventResponse = new RaiseEventResponse(this.mock, ev, null, args);
+			}
+			else
+			{
+				this.raiseEventResponse = null;
+			}
+
 			return this;
 		}
 
 		public override string ToString()
 		{
 			var message = new StringBuilder();
 
-			if (this.FailMessage != null)
+			if (this.failMessage != null)
 			{
-				message.Append(this.FailMessage).Append(": ");
+				message.Append(this.failMessage).Append(": ");
 			}
 
-			var lambda = SetupExpression.PartialMatcherAwareEval().ToLambda();
+			var lambda = this.originalExpression.PartialMatcherAwareEval().ToLambda();
 			var targetTypeName = lambda.Parameters[0].Type.Name;
 
 			message.Append(targetTypeName).Append(" ").Append(lambda.ToStringFixed());
 
-			if (TestMethod != null && FileName != null && FileLine != 0)
+#if !NETCORE
```

Comment:
Now that the fields are removed when targeting .NET Core, we need this directive to completely omit this piece of code.